### DISCUSSION TITLE
Welcome to Thunks

### DISCUSSION THREAD

**Discussion by fogus** [fogus/thunks#1] [Announcements]
Welcome to my little place to share what I'm studying and thinking about and perhaps engaging in thoughtful discussion around that. That said, I will not tolerate assholes. Don't be one.

I wish to provide a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.

As far as topics go I will probably jump all over the place but more often than not I'll land on one or more of the following topics:

* Software -- especially programming languages and systems
* Books and other reading materials
* Clojure, ClojureScript, and their ecosystems
* Games
* Retrocomputing
* Life in general

You don't need to agree, but please disagree in constructive ways.

**Comment by vlaaad** [comment]
> I wish to avoid a harassment-free experience

oh no

**Reply by fogus** [reply]
argh! fixed. thank you.

**Comment by KnowledgeGarden** [comment]
Serious Games, Clojure, Sensemaking, Wicked Issues: they all come together. Add to that: Problem-based Learning (PBL). In my view, that's _Thunk-central_.

**Comment by stathissideris** [comment]
I'm excited about this!

**Comment by zeeshanlakhani** [comment]
would love to be a part of this.
